Absenteeism plots added

GauriSaran opened the pull request:
The plots for absenteeism added in file: index-absentee.Rmd

Following files changed/ added:
        new file:   Data/Historic Projections/historic_cases_projections_twoweekCovidEmployeeData.RData
        new file:   Data/Observed data/cumulative_covid_absenteeism.RData
        modified:   Data/translations/Covid Localization - Home Page.csv
        modified:   R/epinow_functions.R
        modified:   R/observed_data.R
        new file:   content/en/index-absentee.Rmd
        modified:   content/en/wastewater.Rmd

rvyuha commented on R/observed_data.R:
```diff
@@ -968,16 +969,27 @@ reworked_figure <-
     if(isTRUE(date_constraint)){
-      if(as.Date(first(data$date)) > as.Date(last(data$date) - 40)){
+      print("DATE CONST")
+      print(as.Date(last(data$date)) - 40)
```
Avoid hardcoding numbers. I noticed you passing the "interval_num" which is also 40 if these numbers are linked the variable should be used

DougManuel reviewed (commented):
I haven't been able to build the new page but other pages seem to build OK. (I'll look at the build issues and get back if needed).
The other pages did have what looked like print output from R on the page.

GauriSaran commented:
@rvyuha  I have removed the print statements. I forgot to remove them. They were there when I was troubleshooting. They are not needed. Changed variable name a and b. Can you build the index-absentee.Rmd page?

rvyuha commented:
> @rvyuha I have removed the print statements. I forgot to remove them. They were there when I was troubleshooting. They are not needed. Changed variable name a and b. Can you build the index-absentee.Rmd page?

I was at my parents due to construction so had to review on laptop without all the packages will do when I get home

DougManuel commented on Data/translations/Covid Localization - Home Page.csv:
```diff
@@ -27,21 +35,31 @@ Data to {{1}}
 ### Current
 
-{{2}} **{{3}}** days
+{{2}} **{{3}}** days (based on health care worker absenteeism due to COVID-19)
```
Could you delete `(based on health care worker absenteeism due to COVID-19)` and replace with `*`.

Also on line 40.

Then after line 44 put a new line:
`* Based on health care worker absenteeism due to COVID-19.`

DougManuel reviewed (commented):
Can you change the title of the short term projection plots to:
"Projection of hospital census in Ottawa"
"Projection of COVID-19 positive health care workers in Ottawa"

(delete the word "short-term" and other small edits.

DougManuel commented on absenteeism.Rmd:
```diff
@@ -0,0 +1,112 @@
+
+    -   Y-axis: "Cumulative health care worker cases (per 1000 people)
+
+    -   Note: The cumulative health care worker cases (per 1000 people) is the cumulative incidence or total proportion of health care workers who reported a positive PCR or RAT for selected hospitals and health care organizations since December 26, 2021. These are people who notified Occupational Health of their positive result. Based on an estimated reporting rate of 50%. The actual rate per 10 ...
```
Delete, "Based on an estimated reporting rate of 50%. The actual rate per 1000 people 50% lower than reported in this figure..." and replace with,

"Observed COVID-19 cases are adjusted for under-reporting for staff working from home and not providing direct patient care."

DougManuel replied:
Could you modify the same text in the other figures?

DougManuel reviewed (changes requested):
A few suggested text changes and then ready to close the PR.
